build.gradle.kts

```diff
@@ -56,6 +57,20 @@ allprojects {
     }
 }
 
+subprojects {
+    if (!this.name.contains("sample") && this.name != "sentry-test-support") {
+        apply<DistributionPlugin>()
+
+        configure<DistributionContainer> {
+            this.getByName("main").contents {
+                from("build/libs")
+                from("build/publications/maven")
+            }
+        }
+        tasks.named("distZip").dependsOn("publishToMavenLocal")
```

Comment:
Can we amend the task to do a verification like make sure the artifacts were indeed created and they are not 0 bytes or something. At least a smoke test. Validating for the correct content could be not worth the complexity.

Reply:
👍